cartofacade/capi.go

```diff
@@ -323,6 +324,7 @@ func toLidarConfig(lidarConfig LidarConfig) (C.viam_carto_LIDAR_CONFIG, error) {
 func getConfig(cfg CartoConfig) (C.viam_carto_config, error) {
 	vcc := C.viam_carto_config{}
 	vcc.camera = goStringToBstring(cfg.Camera)
+	vcc.movement_sensor = goStringToBstring(cfg.MovementSensor)
```

Comment:
did you figure out what was missing here?

Reply:
Yep! Had to compile everything by running make build